doc/source/getting_started/introduction.ipynb

```diff
@@ -0,0 +1,496 @@
+    "\n",
+    "But first, we set the names of boroughs as an index, to make the results easier to read."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "gdf = gdf.set_index(\"BoroName\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "gdf[\"area\"] = gdf.area\n",
+    "gdf[\"area\"]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Getting polygon boundary and centroid\n",
+    "\n",
+    "To get just the boundary of each polygon (LineString), we can call `GeoDataFrame.boundary`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "gdf['boundary'] = gdf.boundary\n",
+    "gdf['boundary']"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Since we have saved boundary as a new column, we now have two geometry columns in the same `GeoDataFrame`.\n",
+    "\n",
+    "We can also create new geometry, which could be, for example, a buffered version of the original one (i.e., `GeoDataFrame.buffer(10)`) or its centroid:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "gdf['centroid'] = gdf.centroid\n",
+    "gdf['centroid']"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Measuring distance\n",
+    "\n",
+    "We can also measure how far is each centroid from the first one."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "first_point = gdf['centroid'].iloc[0]\n",
+    "gdf['distance'] = gdf['centroid'].distance(first_point)\n",
+    "gdf['distance']"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Making maps\n",
+    "\n",
+    "GeoPandas can also plot maps, so we can check how our geometries look like in space. The key method here is `GeoDataFrame.plot`. In the example below, we plot the `\"area\"` we measured earlier using the active geometry column. We also want to show a legend (`legend=True`)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "gdf.plot(\"area\", legend=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Switching the active geometry (`GeoDataFrame.set_geometry`) to centroids, we can plot the same data using point geometry."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "gdf = gdf.set_geometry(\"centroid\")\n",
+    "gdf.plot(\"area\", legend=True)"
```

Comment:
Another option could also be to chain the set_geometry and plot call (`gdf.set_geomtry(..).plot(..)`), then you don't need to switch back below. 

(but not sure that's necessarily easier to understand for newcomers)

Reply:
Chaining may be good to add to plotting guide. I tried to be super explicit here to make it easier to understand what is happenning.